go.mod

```diff
@@ -32,11 +32,14 @@ require (
 	k8s.io/apiserver v0.28.9
 	k8s.io/client-go v0.28.9
 	k8s.io/kube-aggregator v0.28.9
+	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2 // indirect
 	sigs.k8s.io/controller-runtime v0.15.3
 	sigs.k8s.io/kind v0.17.0 // Do not remove, not used by code but used by build
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/openshift/client-go v0.0.0-20200521150516-05eb9880269c
```

Comment:
can we put this in the main require?

Actually, looks like we have two require sections here - not sure why!